### ISSUE TITLE
[Roots 2] Dedicated server crashes on startup

### ISSUE THREAD

**Issue by Febilian** [RootsTeam/Roots#182]
https://gist.github.com/Febilian/aae70b8f1cb057a722b061c79df03c93

If this is an error on my part I apologize. It mentions this mod specifically so I thought I should bring it up in the chance that it's something that needs to be fixed.

**Comment by elucent** [comment]
This is happening every time you start the server? I can't replicate it with Roots alone. Maybe _try_ removing FoamFix or switching to FoamFix Lawful? I'm not sure it would help, in fact I doubt it, but it's worth a shot and would help me diagnose the issue.

**Comment by Febilian** [comment]
Yes, I had the server set to auto-restart if it crashed, and it was stuck in an endless loop of restarting and crashing until I turned that off.

I'll try reinstalling in a bit here to see if it happens without foamfix.

**Comment by laserlemons** [comment]
Also getting this.

**Comment by Febilian** [comment]
After toying around with a different set of mods and a newer Forge version, this isn't happening for me anymore. I can only assume it is some sort of bizarre mod conflict - though of which ones, I couldn't say - or why it's making _Roots 2_ throw the error.
